res/themes/dark/css/_dark.scss

```
@@ -258,6 +258,7 @@ $composer-shadow-color: rgba(0, 0, 0, 0.28);
 // markdown overrides:
 .mx_EventTile_content .markdown-body pre:hover {
     border-color: #808080 !important; // inverted due to rules below
+    scrollbar-color: auto;
```

Instead of letting the browser do whatever it likes via `auto`, let's try to leverage our theme colours. Because of the inversion filter below, we need to specify the _light_ theme scrollbar colours here, so perhaps something like:

```suggestion
    scrollbar-color: rgba(0, 0, 0, 0.2) transparent; // copied from light theme due to inversion below
```